plugins/mimecast/komand_mimecast/tasks/monitor_siem_logs/task.py

```diff
@@ -67,6 +70,11 @@ def run(  # pylint: disable=unused-argument # noqa: MC0001
                 # we know that there are no more pages to be consumed.
                 has_more_pages = IS_LAST_TOKEN_FIELD not in headers
 
+                # if we have reached the last page then we have caught up to `now`
+                # as result we now then want to use a longer cut off incase the task gets stopped
+                if headers.get(IS_LAST_TOKEN_FIELD) and state.get("normal_running_cutoff") is None:
```

Comment:
worth adding a log in here to say 'Caught up, saving 'normal_running_cutoff' as True' to the state? Should help us once we release and see new set ups that it gets added when we expect?

Reply:
A logger message has been added to reflect this